src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -39,18 +39,41 @@ String privacySettingButtonValue = (Boolean) request.getAttribute("isPrivate")?
     }
   </style>
 
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"> </script>
+
   <script>
+
+    //open add user box
+    $(document).ready(function(){
+      $("#adder").click(function() {
```

Comment:
Notice the `#addr` selector is evaluated on the entire document. If another element has the same name, which is general enough, it'll cause trouble. Let's make these names more specific? Maybe "addUsersModalTrigger" or something.